extensions/arc/.vscodeignore

```diff
@@ -0,0 +1,6 @@
+.gitignore
+instructions.txt
+src/**
```

Comment:
You should be ignoring out/** since you're webpacking it.

Why do you need to run vsce locally? You should be launching ADS from sources which will pick up the extentions automatically

Reply:
I'll fix this.  I wanted people to be able to test the extension without building ADS from source, so I made some changes that allowed me to 'vsce package' it into a .vsix.  I didn't need to push those changes though.